app/controllers/permissions_controller.rb

```diff
@@ -0,0 +1,27 @@
+class PermissionsController < ApplicationController
+  def new
+    @users = User.all
+    @documents = Document.all
+    render :new
+  end
+    
+  def create
+    @permission = Permission.create(permission_params)
+    puts permission_params 
+    if @permission.persisted?
+      redirect_to @permission
```

Comment:
This redirects to /permissions/<id> which is permissions#show. So you need a show action for this controller :)

Reply:
str8 savage